blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/BluefloodServiceStarterTest.java

```diff
@@ -0,0 +1,191 @@
+package com.rackspacecloud.blueflood.service;
+
+import static junit.framework.Assert.*;
+
+import org.junit.*;
+
+import java.io.IOException;
+
+public class BluefloodServiceStarterTest {
+
+    @Before
+    public void setUp() throws IOException {
+
+        Configuration config = Configuration.getInstance();
+
+        config.init();
+
+        config.setProperty(CoreConfig.ZOOKEEPER_CLUSTER, "NONE");
+
+        config.setProperty(CoreConfig.INGEST_MODE, "false");
+        config.setProperty(CoreConfig.ROLLUP_MODE, "false");
+        config.setProperty(CoreConfig.QUERY_MODE, "false");
+
+        BluefloodServiceStarter.SkipInstantiateRestartGauge = true;
```

Comment:
That's why I turned parallelization off. There were tests already modifying configuration in parallel before I even started this branch. We shouldn't turn parallel test execution back on until we can isolate the components better.